charts/camunda-platform/templates/camunda/_helpers.tpl

```diff
@@ -486,6 +486,14 @@ Zeebe templates.
   {{- end -}}
 {{- end -}}
 
+{{/*
+[camunda-platform] Zeebe Gateway GRPC external URL.
+*/}}
+{{- define "camundaPlatform.zeebeGatewayGRPCExternalURL" }}
+  {{ $proto := ternary "https" "http" .Values.zeebeGateway.ingress.grpc.tls.enabled -}}
+  {{- printf "%s://%s%s" $proto .Values.zeebeGateway.ingress.grpc.host .Values.zeebeGateway.contextPath -}}
```

Comment:
:x: gRPC does not support context paths unfortunately :( This is likely what's causing the failure in the integration tests.

Reply:
Dang, you caught me using copy/paste! good catch.